src/promptflow/promptflow/_cli/_pf/entry.py

```diff
@@ -66,13 +66,19 @@ def run_command(args):
             dispatch_experiment_commands(args)
     except KeyboardInterrupt as ex:
         logger.debug("Keyboard interrupt is captured.")
+        # raise UserErrorException(error=ex)
+        # Cant't raise UserErrorException due to the code exit(1) of promptflow._cli._utils.py line 368.
         raise ex
     except SystemExit as ex:  # some code directly call sys.exit, this is to make sure command metadata is logged
         exit_code = ex.code if ex.code is not None else 1
         logger.debug(f"Code directly call sys.exit with code {exit_code}")
+        # raise UserErrorException(error=ex)
+        # Cant't raise UserErrorException due to the code exit(1) of promptflow._cli._utils.py line 368.
```

Comment:
why this? the exception_handler is not used here?

Reply:
This will cause many of our **cli commands** to be unusable, and will directly **exit the program** when using the cli command, because we have a judgment:
![image](https://github.com/microsoft/promptflow/assets/23182548/82aaaf0f-8d7e-4d8b-81b0-378463b405aa)